res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1036,8 +1087,8 @@ NumarkMixtrack3.initButtonsObjects = function() {
         NumarkMixtrack3.decks["D"+i].ShiftButtonControl =
             new SingleDoubleBtn(NumarkMixtrack3.OnShiftButton);
         NumarkMixtrack3.decks["D"+i].ShiftedPFLButtonControl =
-            new SingleDoubleBtn(NumarkMixtrack3.OnShiftedPFLButton);
-    }
+            new SingleDoubleBtn(NumarkMixtrack3.OnShiftedPFLButton);       
+    }    
```

Comment:
Thanks for this tip! I found the option in Notepad++ to show spaces, tabs and all characters